flake.nix

```diff
@@ -15,13 +15,16 @@
   };
 
   inputs = {
-    nixpkgs.url = "github:nixos/nixpkgs/nixos-23.05";
+    #nixpkgs.url = "github:nixos/nixpkgs/nixos-23.05";
+    #BUG: https://github.com/NixOS/nixpkgs/issues/235179
+    #BUG: https://github.com/NixOS/nixpkgs/issues/235526
+    nixpkgs.url = "github:nixos/nixpkgs/nixos-22.11";
     flake-utils.url = "github:numtide/flake-utils";
     nixos-generators = {
       url = "github:nix-community/nixos-generators";
       inputs.nixpkgs.follows = "nixpkgs";
     };
-    nixos-hardware.url = "github:nixos/nixos-hardware";
+    nixos-hardware.url = "github:codehub8/nixos-hardware";
```

Comment:
Please link the pending PR here in the comments with `#TODO: https://github.com/NixOS/nixos-hardware/pull/612`

Reply:
I have listed justification for a forked repository in the file.